src/router.js

```diff
@@ -174,6 +176,18 @@ const routes = [
     name: 'nav.request_access',
     meta: { layout: 'unauthenticated', noAuth: true },
   },
+  {
+    path: '/password/new',
+    component: RequestPasswordReset,
+    name: 'nav.request_password_reset',
+    meta: { layout: 'unauthenticated', noAuth: true },
+  },
+  {
+    path: '/password/reset/:token',
+    component: ResetPassword,
+    name: 'nav.reset_password',
+    meta: { layout: 'unauthenticated', noAuth: true },
+  },
```

Comment:
just organized (most) of the routes to subpages, cuz it was gettin ugly. 

mind rebasing and moving this component to `@/pages/unauthenticated/` and moving the routes to `@/pages/unauth../routes.js`

thanks

Reply:
@erics2783 -- still need this also, ur routes got deleted on removed on rebase :(

Looks good to go tho once the routes are back, yea?